maintainers/maintainer-list.nix

```
@@ -6684,6 +6684,13 @@
       fingerprint = "932F 3E8E 1C0F 4A98 95D7  B8B8 B0CA A28A 0295 8308";
     }];
   };
+  kenran_ = {
```

```suggestion
  kenran = {
```
I don't think we need the extra underline.